docs/cadence/anti-patterns.mdx

```diff
@@ -70,26 +69,6 @@ rather than inside contract utility functions.
 There are some scenarios where using an `AuthAccount` object is necessary, such as a cold storage multi-sig,
 but those cases are extremely rare and `AuthAccount` usage should still be avoided unless absolutely necessary.
```

Comment:
Maybe replace `AuthAccount` with `auth(...) &Account`

```suggestion
but those cases are extremely rare and `auth(...) &Account` usage should still be avoided unless absolutely necessary.
```

Reply:
Fixed in 7df3a78d27dc6f265ca7530c9bdae4370018d083